hw/syn/tools/dc/testsynth.tcl

```diff
@@ -0,0 +1,113 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Simple tcl script for DC to do some wire-load-model-based test syntheses.
+
+#####################
+##  PREPARE FLOW   ##
+#####################
+
+# tool setup
+source ../../../foundry/syn/dc/setup.tcl
+
+# paths
+set WORKLIB  "WORK"
+set REPDIR   "REPORTS"
+set DDCDIR   "DDC"
+set VLOGDIR  "NETLISTS"
+
+exec mkdir -p ${REPDIR} ${DDCDIR} ${VLOGDIR} ${WORKLIB}
+
+# define work lib path
+define_design_lib WORK -path $WORKLIB
+
+#######################
+##  DESIGN SOURCES  ###
+#######################
+
+set DUT "rv_plic_target"
+
+lappend search_path "../../../ip/prim/rtl/"
+set SRC {  "../../../ip/rv_plic/rtl/rv_plic_target.sv" }
```

Comment:
as before, is SRC overridden in the generic case?

Reply:
same answer ;).

Reply:
ok, for all of these I wasn't sure how generic it was supposed to be, or if it was an example to be modified by the designer for their own usage.  I got it now :)